components/Navbar.tsx

```diff
@@ -36,21 +36,19 @@ const Navbar: React.FC<NavbarProps> = ({ onAsanaClick }) => {
         <img src="/assets/images/home.png" alt="Home" />
         Introduction
       </Link>
-      <div className='flex'>
+      <div className="flex">
         <input type="search" placeholder="search" className={styles.search} />
-        <img src="/assets/images/heart.png" alt="heart" className={styles.heart}/>
+        <img src="/assets/images/heart.png" alt="heart" className={styles.heart} />
       </div>
-      
-      {asanas.map(asana => (
+
+      {asanas.map((asana: any) => (
```

Comment:
The better approach is to avoid `any` in our code. You can define the type of Asana in a file (usually `types.ts` and import it anywhere you want. that type can be used here and in `AsanaPreview` component and ...)